notebooks/1.0-vse-eda-rooms.ipynb

```diff
@@ -0,0 +1,614 @@
+{
```

Comment:
<p>The first sentence it's true, but I don't agree with the second. The analyses cant give us the final answer about feature selection for the model, but it's a valuable information</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>